### ISSUE TITLE
post window transparent text

### ISSUE THREAD

**Issue by bitwo** [supercollider/supercollider#2835]
Bug description:
The text in  the post window is mostly transparent and unreadable on a Lenovo IdeaPad 100 ubuntu studfio and a Medion PC ubuntu studio. Some other machines ubuntu studio perform well.  A skilled long term user of supercollider did weather find  a reason nor solution. 
Steps to reproduce:
1. fresh installation of supercollider 3.9dev from start on 
2.
3.

Actual result:

Expected result:

**Comment by brianlheim** [comment]
Could you post a screenshot of what you're seeing? And could someone else please try to reproduce? @bagong @snappizz ? I feel like this would come up more often if it were a reproducible issue.

**Comment by snappizz** [comment]
Is the code editor experiencing the same issue?

**Comment by bitwo** [comment]
The code editor is o. k.
The reason for this misbehavour I found with linux themes. Some of the 
(many downloadable) themes "wipe out" the text in the post window. My 
explanation I generated for myself for understanding: Some of the themes 
use some pixel resolution that do not integer fit with the post window. 
Alternatively one color bit may not cooperate? not sure. Sorry, that is 
obviously not an error of supercollider but improper pairing.

PS: I usually change the theme because the original theme of the linux 
distro (ubuntu studio) does not provide arrows in the scroll bar. And I 
change it often depending on the environment conditions of illumination.

Am 15.04.2017 03:07, schrieb Nathan Ho:
